### ISSUE TITLE
[i3fyra][i3king] windows spawned from config file, is not placed in layout

### ISSUE THREAD

**Issue by budRich** [budlabs/i3ass#184]
mentioned by @1ntronaut in #180 .

>via regular i3config, I do launch i3term instances (running a slightly modified i3info script, as well as starting i3king --verbose) as to monitor i3fyra/i3king activity. On startup, these windows launch as if they would in regular/default i3 ; they 1) do not create i3fyra containers (unless I untile + retile them manually); 2) do not adhere to $layout on launch; and 3) won't be ruled by i3king and thus will not move to their specified containers.
I always have to manually i3fyra --float these windows twice (creating the 'A' container) and manually i3viswiz l|d|u|r them into their supposed containers (B+D). I also have to spawn 2 additional windows and move them around to be able to attain my preferred layout ; with 4 visible containers (i3list[LVI]), in the correct layout. I do this every time I login/start up i3, after which it will work. (P.S. i3viswiz moving does the trick. i3fyra -m A|B|C|D does NOT.. and also won't move windows to containers if I don't i3viswiz move them manually to create ABCD containers beforehand.

@1ntronaut , can you provide a minimal i3config and i3king rules file that replicates this behavior?

**Comment by 1ntronaut** [comment]
Hi @budRich ;
I will try to reproduce this and provide you with the files. I'll try on my current install, but I might end up not being able to, and if so, it could take a while.

**Comment by budRich** [comment]
@1ntronaut : is this still an issue?

**Comment by 1ntronaut** [comment]
@budRich hi bud. Darn, I can't believe I'd never commented/replied in order to answer/close this issue.

With correctly configured i3king rules file, everything except the auto-started i3term running 'i3king --verbose' (w/ urxvtc) is automatically put in the correct i3fyra container, as long as i3king is running.
- - - -
tl;dr
Everything works as is intended, **NOT** an issue anymore
